logfire/_internal/main.py

```diff
@@ -1174,17 +1174,26 @@ def instrument_httpx(
         self._warn_if_not_initialized_for_instrumentation()
         return instrument_httpx(self, client, **kwargs)
 
-    def instrument_celery(self, **kwargs: Unpack[CeleryInstrumentKwargs]) -> None:
+    def instrument_celery(self, **kwargs: Any) -> None:
```

Comment:
What about `skip_dep_check`?

Reply:
I removed on purpose. Is it really important? People shouldn't use it anyway... Unless specific cases like `psycopg-binary`...

Reply:
No it's not that important but I wasn't sure if it was intentional. Let's leave it.